pkg/kubectl/cmd/util/factory_object_mapping.go

```diff
@@ -96,6 +97,9 @@ func (f *ring1Factory) objectLoader() (meta.RESTMapper, runtime.ObjectTyper, err
 	// anything.  This mirrors old behavior that used to fallback to a hardcoded list of API types compiled
 	// into kubernetes.
 	if err != nil {
+		if _, ok := err.(net.Error); ok {
```

Comment:
checking for a particular error type here is extremely fragile with go's type system... wrapping errors would make this fail. Need a better way to check this, and a test exercising it if we want it to stick. I also think we'd still want to continue if we got any results (if `len(groupResources) > 0`)